math/fibonacci.cpp

```diff
@@ -17,8 +17,9 @@
 unsigned int fibonacci(unsigned int n) {
```

Comment:
shall we take this opportunity to update the doc for this function by adding `@param` and `@returns`?

Reply:
done